docs/source/tutorials/how_to_parametrize_a_task.rst

```diff
@@ -17,100 +17,41 @@ An example
 We reuse the previous example of a task which generates random data and repeat the same
```

Comment:
Link that to make it more self-contained. I would think that 98% of readers won't read this like a book, but arrive at this page because they have some issue.